onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -678,13 +678,13 @@ Status QnnBackendManager::SetHtpPowerConfig(uint32_t htp_power_config_client_id,
   dcvs_v3.setSleepDisable = 0;
   dcvs_v3.sleepDisable = 0;
   dcvs_v3.setDcvsEnable = 1;
-  dcvs_v3.dcvsEnable = kDcvsDisable;
   dcvs_v3.powerMode = QNN_HTP_PERF_INFRASTRUCTURE_POWERMODE_PERFORMANCE_MODE;
```

Comment:
Just a note -- this can be set to power_saver_mode for ExtremeLowPower mode.

Reply:
Done. Thanks!